magenta/interfaces/midi/midi_hub.py

```diff
@@ -148,6 +148,7 @@ class Metronome(threading.Thread):
     velocity: The velocity of the metronome's tick `note_on` message.
     pitch: The pitch of the metronome's tick `note_on` message.
     duration: The duration of the metronome's tick.
+    qpb: The number of quarter notes per bar.
```

Comment:
not sure what you want it after, but I moved it up now that it is a non default argument. Where is the best place to put it?